packages/thirdweb/src/react/web/ui/ConnectWallet/Details.tsx

```diff
@@ -1083,12 +1089,32 @@ function InAppWalletUserInfo(props: {
         "../../../../wallets/in-app/web/lib/auth/index.js"
       );
 
+      const isInAppWallet =
+        adminWallet &&
+        (adminWallet.id === "inApp" || adminWallet.id.startsWith("ecosystem."));
+
+      if (!isInAppWallet) {
+        return null;
+      }
+
+      let ecosystem: Ecosystem | undefined;
+      if (adminWallet.id.startsWith("ecosystem.")) {
```

Comment:
We have an `isEcosystemWallet` util for this

Reply:
nice, used that